src/templates/Handbook.tsx

```diff
@@ -410,6 +422,27 @@ export default function Handbook({
                                                             Edit this page
                                                         </Link>
                                                     )}
+                                                    {rawBody && (!hideLastUpdated || filePath) && (
+                                                        <span className="text-primary/30 dark:text-primary-dark/30">
+                                                            |
+                                                        </span>
+                                                    )}
```

Comment:
These checks are getting kinda complex. We should probably just use a `gap` CSS property to draw `|` in between the other components - or if we wanna be old-fashioned, some border-lefts